src/filters/index.js

```diff
@@ -8,6 +8,15 @@ module.exports = function (fractal) {
         field_value(str) {
             return str;
         },
+        without(element, exclude_elements) {
+            filtered_element = element;
```

Comment:
In JavaScript, variable assignments are always by reference, so this does not copy the value (like in PHP).  You need to clone the object instead.  There are also (deep)clone helper functions on lodash.

Reply:
great, pushed the update!